dist/pouchdb.quick-search.js

```diff
@@ -275,6 +275,7 @@ exports.search = utils.toPromise(function (opts, callback) {
         docIdsToFieldsToQueryTerms, docIdsToFieldsToNorms, fieldBoosts);
       return rows;
     }).then(function (rows) {
+      total_rows = rows.length;
```

Comment:
Oh wait, I'm dumb, you're right. This is exactly what we want `total_rows` to report - the total number of possible results for this query, before any slicing occurs due to offset/limit. Great work. :)